# error

#error handling (hatayı ele alma)
try:
    x = int(input("x: "))
    y = int(input("y: "))
    print(x+y)
except:
    print("hata oluştu")